<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WFileTag</class>
 <widget class="QWidget" name="WFileTag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,13">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="pathLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>13</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #7598db;  /* 设置字体颜色为橙色 */
    background-color: #e0e0e0;  /* 背景颜色为灰色（可选） */
    border: 1px solid #ccc;  /* 边框颜色为浅灰色（可选） */
    border-radius: 5px;  /* 设置圆角（可选） */
    padding: 5px 10px;  /* 设置内边距（可选） */
}
</string>
         </property>
         <property name="text">
          <string> 跳转</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
